core/src/test/java/org/elasticsearch/index/query/RangeQueryBuilderTest.java

```diff
@@ -93,32 +89,16 @@ protected RangeQueryBuilder doCreateTestQueryBuilder() {
     }
 
     @Override
-    protected Query doCreateExpectedQuery(RangeQueryBuilder queryBuilder, QueryParseContext context) throws IOException {
-        Query expectedQuery;
-        String fieldName = queryBuilder.fieldName();
-        if (getCurrentTypes().length == 0 || (fieldName.equals(DATE_FIELD_NAME) == false && fieldName.equals(INT_FIELD_NAME) == false)) {
-            expectedQuery = new TermRangeQuery(fieldName, BytesRefs.toBytesRef(queryBuilder.from()),
-                    BytesRefs.toBytesRef(queryBuilder.to()), queryBuilder.includeLower(), queryBuilder.includeUpper());
-
-        } else if (fieldName.equals(DATE_FIELD_NAME)) {
-            DateMathParser forcedDateParser = null;
-            if (queryBuilder.format() != null) {
-                forcedDateParser = new DateMathParser(Joda.forPattern(queryBuilder.format()));
-            }
-            DateTimeZone dateTimeZone = null;
-            if (queryBuilder.timeZone() != null) {
-                dateTimeZone = DateTimeZone.forID(queryBuilder.timeZone());
-            }
-            MappedFieldType mapper = context.fieldMapper(queryBuilder.fieldName());
-            expectedQuery = ((DateFieldMapper.DateFieldType) mapper).rangeQuery(BytesRefs.toBytesRef(queryBuilder.from()), BytesRefs.toBytesRef(queryBuilder.to()),
-                    queryBuilder.includeLower(), queryBuilder.includeUpper(), dateTimeZone, forcedDateParser);
+    protected void doAssertLuceneQuery(RangeQueryBuilder queryBuilder, Query query, QueryParseContext context) throws IOException {
+        if (getCurrentTypes().length == 0 || (queryBuilder.fieldName().equals(DATE_FIELD_NAME) == false && queryBuilder.fieldName().equals(INT_FIELD_NAME) == false)) {
+            assertThat(query, instanceOf(TermRangeQuery.class));
+        } else if (queryBuilder.fieldName().equals(DATE_FIELD_NAME)) {
+            //we can't properly test unmapped dates because LateParsingQuery is package private
```

Comment:
Should we increase visibility in this case? Makes only sense if we can then really test more, though, otherwise okay to leave it I guess.